.env.template

```diff
@@ -14,3 +14,5 @@ REACT_APP_FIREBASE_MEASUREMENT_ID=VAR_REACT_APP_FIREBASE_MEASUREMENT_ID
 REACT_APP_DID_KEY_VERSION=VAR_REACT_APP_DID_KEY_VERSION
 REACT_APP_VERSION=$npm_package_version
 REACT_APP_DISPLAY_CONSOLE=VAR_REACT_APP_DISPLAY_CONSOLE
+REACT_APP_WEBAUTHN_ORIGIN=VAR_REACT_APP_WEBAUTHN_ORIGIN
+REACT_APP_WEBAUTHN_RPID=VAR_REACT_APP_WEBAUTHN_RPID
```

Comment:
Where do these `VAR_` values come from?

Reply:
These VAR_ values are using for deployment.

Reply:
Yeah, I meant where they are defined, because I couldn't find `VAR_` anywhere in the repos. So I was wondering how I would set these locally. But now I remember this is just the template file and I have a separate `.env` file in my local environment. Nevermind!

Reply:
Oh, I guess these and their defaults should be added to the README though?

Reply:
Yes, you are correct.